server/utils.js

```diff
@@ -1,7 +1,7 @@
 const terms = require('kth-canvas-utilities').terms
-const Promise = require('bluebird') // use bluebird to get a little more promise functions then the standard Promise AP
 const { unlink } = require('fs')
-const unlinkAsync = Promise.promisify(unlink)
+const util = require('util')
```

Comment:
What is this `util`?  I don't see a `util.js` and there is no direct dependency with that name.  Where should I look?

Reply:
It's from the standard node library =)